marklogic-data-hub-central/ui/src/components/raw-data/raw-data-card-view/raw-data-card-view.test.tsx

```diff
@@ -28,5 +29,30 @@ describe("Raw data card view component", () => {
         expect(getByTestId('/Customer/Cust3.json-InfoIcon')).toBeInTheDocument();
         expect(getByTestId('/Customer/Cust3.json-sourceFormat')).toBeInTheDocument();
         expect(getByTestId('/Customer/Cust3.json-detailViewIcon')).toBeInTheDocument();
+
+        //verify tooltips 
+        fireEvent.mouseOver(getByTestId('/Customer/Cust1.json-URI'))
+        await waitForElement(() => getByText('/Customer/Cust1.json'));
+
+        fireEvent.mouseOver(getByTestId('/Customer/Cust1.json-InfoIcon'))
+        await waitForElement(() => getByText('View info'));
+
+        fireEvent.mouseOver(getByTestId('/Customer/Cust1.json-detailViewIcon'))
+        await waitForElement(() => getByText('Detail view'));
+
+        //verify snippet content for json/xml/text docs
+        expect(getByTestId('/Customer/Cust1.json-snippet')).toBeInTheDocument();
```

Comment:
This line seems redundant because the next line, where we are trying to access the 'textContent' for the same element, does the needful and will definitely fail if the element is not available in the document.
The same goes for the lines 47 and 50.

Reply:
Fixed. Thanks.